api/v1alpha1/xlinecluster_types.go

```diff
@@ -47,6 +49,43 @@ type XlineClusterList struct {
 	Items           []XlineCluster `json:"items"`
 }
 
+type XlineArgs struct {
+	JaegerOffline               bool    `json:"jaeger-offline,omitempty"`
+	JaegerOnline                bool    `json:"jaeger-online,omitempty"`
+	JaegerLevel                 bool    `json:"jaeger-level,omitempty"`
+	ClientUseBackoff            bool    `json:"client-use-backoff,omitempty"`
+	AuthPrivateKey              *string `json:"auth-private-key,omitempty"`
+	AuthPublicKey               *string `json:"auth-public-key,omitempty"`
+	JaegerOutputDir             *string `json:"jaeger-output-dir,omitempty"`
+	LogFile                     *string `json:"log-file,omitempty"`
+	LogRotate                   *string `json:"log-rotate,omitempty"`
+	LogLevel                    *string `json:"log-level,omitempty"`
+	HeartbeatInterval           *string `json:"heartbeat-interval,omitempty"`
+	ServerWaitSyncedTimeout     *string `json:"server-wait-synced-timeout,omitempty"`
+	RetryTimeout                *string `json:"retry-timeout,omitempty"`
+	RpcTimeout                  *string `json:"rpc-timeout,omitempty"`
+	BatchTimeout                *string `json:"batch-timeout,omitempty"`
+	ClientWaitSyncedTimeout     *string `json:"client-wait-synced-timeout,omitempty"`
+	ClientProposeTimeout        *string `json:"client-propose-timeout,omitempty"`
+	ClientInitialRetryTimeout   *string `json:"client-initial-retry-timeout,omitempty"`
+	ClientMaxRetryTimeout       *string `json:"client-max-retry-timeout,omitempty"`
+	GcInterval                  *string `json:"gc-interval,omitempty"`
+	RangeRetryTimeout           *string `json:"range-retry-timeout,omitempty"`
+	CompactTimeout              *string `json:"compact-timeout,omitempty"`
+	SyncVictimsInterval         *string `json:"sync-victims-interval,omitempty"`
+	WatchProgressNotifyInterval *string `json:"watch-progress-notify-interval,omitempty"`
+	CurpDir                     *string `json:"curp-dir,omitempty"`
+	CompactSleepInterval        *string `json:"compact-sleep-interval,omitempty"`
+	RetryCount                  int     `json:"retry-count,omitempty"`
+	BatchMaxSize                int     `json:"batch-max-size,omitempty"`
+	FollowerTimeoutTicks        int     `json:"follower-timeout-ticks,omitempty"`
+	CandidateTimeoutTicks       int     `json:"candidate-timeout-ticks,omitempty"`
+	LogEntriesCap               int     `json:"log-entries-cap,omitempty"`
+	CmdWorkers                  int     `json:"cmd-workers,omitempty"`
```

Comment:
It's better to use `kubebuilder:validation:Minimum` to make sure all these numeric args are larger than or equal to 1.